physionet-django/console/views.py

```diff
@@ -929,6 +929,35 @@ def users_search(request, group):
     raise Http404()
 
 
+@login_required
+@user_passes_test(is_admin, redirect_field_name='project_home')
+def known_references_search(request):
+    """
+    Search credential applications and user list.
+    """
+
+    if request.method == 'POST':
+        search_field = request.POST['search']
+
+        applications = CredentialApplication.objects.filter(
+            Q(reference_email__icontains=search_field) |
+            Q(reference_name__icontains=search_field) |
+            Q(user__profile__last_name__icontains=search_field) |
+            Q(user__profile__first_names__icontains=search_field))
+
+        all_known_ref = applications.exclude(
+            reference_contact_datetime__isnull=True).order_by(
+            '-reference_contact_datetime')
+
+        if len(applications) == 0:
+            all_known_ref = paginate(request, applications, 50)
+
+        return render(request, 'console/application_list.html', {
+            'all_known_ref':all_known_ref})
```

Comment:
nitpick but pep8 wants a space after the `:`

Reply:
Fixed in next commit :)